crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -38,12 +38,12 @@
         pass
 
 
-match (  # comment
-    # let's go
-    yield foo
-):  # another comment
-    case ["a", "b", "c"]:
-        pass
+# match (  # comment
+#     # let's go
+#     yield foo
+# ):  # another comment
+#     case ["a", "b", "c"]:
+#         pass
```

Comment:
This should be uncommented before merging

Reply:
Forgot to uncomment this, thanks for the catch!